cmd/controller-manager/app/options/options.go

```diff
@@ -36,6 +36,7 @@ type ControllerManagerServer struct {
 }
 
 const defaultResyncInterval = 5 * time.Minute
+const defaultBrokerRelistInterval = 24 * time.Hour
```

Comment:
It's set to 30 minutes by the helm chart (see `values.yaml`), which is a more reasonable default for development.  When you run the binary, I think the default should be more reasonable for production, hence 24 hours.